docs/snippets/react/button-story.with-hooks.js.mdx

```diff
@@ -0,0 +1,23 @@
+```js
+// Button.stories.js | Button.stories.ts
+
+import React, { useState } from 'react';
+
+import { Button } from './Button';
+
+export const Primary = () => {
+
+  // Sets the hooks for both the label and primary props
+  const [value, setValue] = useState('Secondary');
```

Comment:
is this a pattern we want to encourage? i'd suggest that we want to encourage args for storing state and this would be some advanced usage

Reply:
Just included a couple of commits that should address your comment. Let me know if you're ok with it.